doc/formats.md

```diff
@@ -352,6 +353,16 @@ Return `matroska_path` string for a box decode value
 - https://www.matroska.org/technical/codec_specs.html
 - https://wiki.xiph.org/MatroskaOpus
 
+### mbr
+
+The MBR (Master Boot Record) tells the computer how to find the bootloader.
+It is a fixed 512-byte record that PC motherboards know how to read.
+It contains 446 bytes of executable 16-bit x86 code, and a 64-bit partition
+table, then two magic bytes to signify the end of the record.
+
+#### References and links
+- https://thestarman.pcministry.com/asm/mbr/PartTables.htm#mbr
+
```

Comment:
This has changed a bit in master since the PR, was redone so that documentation can embedded into fq, accessible via `help(...)` and `--help`. Ex how it works now:
https://github.com/wader/fq/blob/master/format/asn1/asn1_ber.jq#L10
https://github.com/wader/fq/blob/master/format/asn1/asn1_ber.go#L40

And then `make doc` will generate things for `formats.md` also

Reply:
I added a `format/mbr/mbr.jq` file with the `_mbr__help` function, and then added `Functions: []string{"_help"}` to the registry.MustRegister call in mbr.go. Before doing this, `make` built the fq binary after all the tests passed. With the _help stuff, it fails with 

```
expr:1:0: compile: function not defined: _mbr__help/0
```

Reply:
yep, that was it